api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -541,6 +541,88 @@ func Convert_v1alpha1_VirtualMachineStatus_To_v1alpha2_VirtualMachineStatus(
 	return nil
 }
 
+func convert_v1alpha2_Conditions_To_v1alpha1_Conditions(conditions []Condition) []Condition {
+	var preReqCond, vmClassCond, vmImageCond, vmSetResourcePolicy, vmBootstrap *Condition
+
+	for i := range conditions {
+		c := &conditions[i]
+
+		switch c.Type {
+		case VirtualMachinePrereqReadyCondition:
+			preReqCond = c
+		case v1alpha2.VirtualMachineClassConditionReady:
```

Comment:
Is this VirtualMachineConditionClassReady instead?.. 
https://github.com/vmware-tanzu/vm-operator/blob/3c428b154b5caea84f2da3ee4b1ff7512ee516f5/api/v1alpha2/virtualmachine_types.go#L18